### PR TITLE
Fix: Discharge patient showing live

### LINKED ISSUE #9289: Discharged Patient status showing as "live"
**Describe the bug**
Discharged patient Status showing as Live for some patients. its happening for the patient having multiple consultations filed. Also it not happening for all the patients

reported by karnataka state official and arike staffs

**To Reproduce**

1. Go to discharged patient on staging website
2. Click on Patient "Raw" or Ritwik Nishad" ( facility - Gayan's Hospital)
3. Go to Patient Details page 

**Expected behavior**
Status should be showing as Discharged for the discharged patients

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by AdityaJ2305**
## Proposed Changes

- Fixes #9289 
- Change 1
- Change 2
- More?

@ohcnetwork/care-fe-code-reviewers

## Screenshot
Screenshot 2024-12-15 at 7 12 36 PM

## Merge Checklist

- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews

## Summary by CodeRabbit

- **New Features**
	- Introduced a new section in the ConsultationCard to display patient status with a visual indicator.

- **Bug Fixes**
	- Enhanced layout organization in the ConsultationCard for improved clarity.

- **Refactor**
	- Removed the patient status display from the Demography component, adjusting the layout accordingly while maintaining core functionalities.

### CODE REVIEW

File: src/components/Facility/ConsultationCard.tsx [hunk 1]
```diff
@@ -53,16 +54,39 @@ export const ConsultationCard = (props: ConsultationProps) => {
...
+            <Chip
+              size="medium"
+              variant="custom"
+              className={
+                !isLastConsultation
+                  ? "bg-blue-100 text-blue-800"
+                  : "bg-red-100 text-red-800"
+              }
+              text={!isLastConsultation ? "LIVE" : "DISCHARGED"}
+            />
```
Comment (rithviknishad) [inline src/components/Facility/ConsultationCard.tsx]: this just checks for whther the encounter is last consultation or not right? and doesn't check if that last consultation is discharged or not.

This is not the right logic.

image

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/components/Facility/ConsultationCard.tsx (hunk 1 above)
Comment (AdityaJ2305) [reply src/components/Facility/ConsultationCard.tsx]: Got it, I’ll make the necessary changes.

**Review by rithviknishad — APPROVED** [review]

**Review by Jacobjeevan — APPROVED** [review]
LGTM, minor thing, but can we switch to using size small, so that it matches with IP/OP encounter chip.

**Comment by AdityaJ2305** [comment]
> LGTM, minor thing, but can we switch to using size small, so that it matches with IP/OP encounter chip.

Yeah
Done

**Comment by nihal467** [comment]
LGTM

**Pull request merged**